src/Microsoft.JSInterop/JSAsyncCallResult.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.JSInterop
+{
+    /// <summary>
+    /// Represents an intermediate value while handling an async call to JavaScript.
+    /// </summary>
+    public readonly struct JSAsyncCallResult
```

Comment:
What's the benefit from making this a struct? won't it always be boxed

Reply:
Good point. Changed it.